gateway-server/src/main/java/org/apache/knox/gateway/websockets/GatewayWebsocketHandler.java

```diff
@@ -114,7 +114,8 @@ public Object createWebSocket(ServletUpgradeRequest req,
       final String path = requestURI.getPath();
 
       /* URL used to connect to websocket backend */
-      final String backendURL = getMatchedBackendURL(path);
+      final String backendURL = getMatchedBackendURL(path, requestURI);
+      LOG.logMessage("Generated backend URL for websocket connection: " + backendURL);
```

Comment:
Use a new function to log generic log messages, `LOG.logMessage` is used to log WS messages.

Reply:
Done. Thanks